.github/workflows/dashboard-docker-tag.yml

```diff
@@ -0,0 +1,29 @@
+name: Kinto Dashboard Docker Image Tag
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  tag:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
@bappr @nandiheath do we need this "uses" line?

Reply:
we don't have it here: https://github.com/kintoproj/kinto-core/blob/main/.github/workflows/push_to_registry.yml#L22

Reply:
no we don't, and we don't need line 16 and 17 too since we are not building anything